cardano-config/src/Cardano/Config/Types.hs

```diff
@@ -199,6 +199,8 @@ instance FromJSON NodeConfiguration where
                               <*> v .:? "TraceTxInbound" .!= False
                               <*> v .:? "TraceTxOutbound" .!= False
                               <*> v .:? "TraceTxSubmissionProtocol" .!= False
+                              <*> v .:? "TraceHandshake" .!= False
```

Comment:
Please put `TraceHandshake` & `TraceLocalHandshake`  in alphabetical order like the rest of the tracers.